CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Change Log
 
+## 10.2.0 (July 11, 2020)
+
+- Enhancement: Add `conf` to `Config` folder. ([@georgeherby](https://github.com/georgeherby) in [#2532](https://github.com/vscode-icons/vscode-icons/pull/2532))
+- Enhancement: Extend `husky` to support `husky.config.js`. ([@rifaidev](https://github.com/rifaidev) in [#2528](https://github.com/vscode-icons/vscode-icons/pull/2528))
+- Enhancement: Update `BLoC` folder icon. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2527](https://github.com/vscode-icons/vscode-icons/pull/2527))
+- Enhancement: Enhance `pinescript` with extra id. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2524](https://github.com/vscode-icons/vscode-icons/pull/2524))
+- Enhancement: Update `Scala` icon. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2522](https://github.com/vscode-icons/vscode-icons/pull/2522))
+- Feature: Support for `ftHTML`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2520](https://github.com/vscode-icons/vscode-icons/pull/2520))
+- Feature: Support for `Bats`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2519](https://github.com/vscode-icons/vscode-icons/pull/2519))
+- Feature: Support for `Drawio`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2518](https://github.com/vscode-icons/vscode-icons/pull/2518))
+- Feature: Support for `glyphs`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2517](https://github.com/vscode-icons/vscode-icons/pull/2517))
+- Feature: Support for `Unity`. ([@shubham-saudolla](https://github.com/shubham-saudolla) in [#2510](https://github.com/vscode-icons/vscode-icons/pull/2510))
+- Feature: Support for `Rmd`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2508](https://github.com/vscode-icons/vscode-icons/pull/2508))
+- Feature: Support for `jscpd`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#2507](https://github.com/vscode-icons/vscode-icons/pull/2507))
+- Feature: Support for `DVC`. ([@shubham-saudolla](https://github.com/shubham-saudolla) in [#2506](https://github.com/vscode-icons/vscode-icons/pull/2506))
+- Enhancement: Extend `zeit` icon to `vercel` files. ([@nvignola](https://github.com/nvignola) in [#2487](https://github.com/vscode-icons/vscode-icons/pull/2487))
```

Comment:
Need to change order place.

Reply:
This is what happens when I don't wear my glasses while preparing the changelog! 😜 Good catch! 🦅